skll/data/readers.py

```diff
@@ -327,6 +340,22 @@ def _parse_dataframe(self, df, id_col, label_col, features=None):
             raise ValueError("No features found in possibly "
                              "empty file '{}'.".format(self.path_or_list))
 
+        if ignore_blanks and replace_blanks_with is not None:
+            raise ValueError("You have set `ignore_blanks=True`, but 'replace_blanks_with' "
+                             "is not `None`; 'replace_blanks_with' can only have a value "
+                             "when 'ignore_blanks' is set to `False`.")
```

Comment:
How about something like:

"You cannot both drop blanks and replace them. 'replace_blanks_with' can only have a value when 'drop_blanks' is `False`."